### PR TITLE
Automate OCP-44790 externalIP-upgrade-test

### PR DESCRIPTION
**Pull request by weliang1**
Automate OCP-44790 externalIP-upgrade-test

Test log:
http://file.rdu.redhat.com/~weliang/externalIP-pre-log
http://file.rdu.redhat.com/~weliang/externalIP-post-log

/cc @openshift/team-sdn-qe

### CODE REVIEW

File: features/upgrade/sdn/externalIP-upgrade.feature [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    And I wait up to 500 seconds for the steps to pass:
+    """
+    When I run oc create over "externalip_service1.json" replacing paths:
+      | ["spec"]["externalIPs"][0] | <%= cb.hostip %>    |
+      | n                          | externalip-upgrade  |
+    Then the step should succeed
+    """
+
+    # Create a pod
+    Given I obtain test data file "networking/externalip_pod.yaml"
```
Comment (asood-rh) [inline features/upgrade/sdn/externalIP-upgrade.feature:30]: @weliang1 If this pod to just to validate the service configured with external ip, is another YAML file (externalip_pod.yaml) needed? Could you not use existing YAML files and replace the pod name to what ever is desired for this test?

Comment (weliang1) [reply features/upgrade/sdn/externalIP-upgrade.feature:30]: @asood-rh Good suggestion, let me update my script to be the way as you suggest here.

**Pull request merged**